docs/platform/manager/third-party-authentication/index.md

```diff
@@ -0,0 +1,60 @@
+---
+author:
+  name: Linode Community
+  email: docs@linode.com
+description: 'This guide shows you how to enable Third Party Authentication (TPA) for your Linode account, allowing you to log in to Linode Cloud Manager with third party credentials.'
+og_description: 'This guide shows you how to enable Third Party Authentication (TPA) for your Linode account, allowing you to log in to Linode Cloud Manager with third party credentials.'
+keywords: ['tpa','third party authentication','github','account','password']
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+published: 2020-04-22
+modified_by:
+  name: Linode
+title: "How to Enable Third Party Authentication on Your Linode Account"
+h1_title: "Enabling Third Party Authentication on Your Linode Account"
+---
+
+Linode Cloud Manager supports Third-Party Authentication (TPA). This allows you to log in to the Linode Cloud Manager with another provider's login credentials.
+
+## Enabling Third Party Authentication
+
+1.  To get started with TPA, log in to the [Linode Cloud Manager](https://cloud.linode.com) using your existing username and password.
+
+1.  Navigate to your profile by clicking on your username in the top right of the screen.
+
+    ![Navigate to your profile by clicking on your username in the top right of the screen](profile-menu.png)
+
+1.  Click on the **Password & Authentication** tab, then click on the **Third Party Authentication** tab.
```

Comment:
```suggestion
1.  Click on the **Password & Authentication** tab, then click on the **Third Party Authentication** sub-tab.
```
"sub-tab"? The picture is useful enough that I'm not terribly concerned with this change.